fourmolu/Dockerfile

```
@@ -0,0 +1,42 @@
+FROM restyled/stack-build-minimal:20.04 as builder
+label maintainer="Tristan de Cacqueray <tdecacqu@redhat.com>"
```

Should've mentioned this in my first review, sorry. I'm also fine changing this myself; just want to be consistent with the other commands throughout the file.

```suggestion
LABEL maintainer="Tristan de Cacqueray <tdecacqu@redhat.com>"
```

---

Done.